src/core/jsonl/include/sourcemeta/core/jsonl.h

```diff
@@ -1,11 +1,11 @@
 #ifndef SOURCEMETA_CORE_JSONL_H_
 #define SOURCEMETA_CORE_JSONL_H_
 
+#include "sourcemeta/core/json_value.h"
```

Comment:
Same as in the other PR. Use `<...>` includes and please keep the ordering. This should be close to the line you removed and not as a standalone thing here